packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -275,6 +285,18 @@ struct WindowData {
             self->m_useDirectDebugger = IsDlgButtonChecked(hwnd, IDC_DIRECTDEBUGGER) == BST_CHECKED;
             self->m_breakOnNextLine = IsDlgButtonChecked(hwnd, IDC_BREAKONNEXTLINE) == BST_CHECKED;
 
+            auto themeComboBox = GetDlgItem(hwnd, IDC_THEME);
+            self->m_theme = ComboBox_GetCurSel(themeComboBox);
+            auto theme;
+            switch (self->m_theme) {
+                // Matches list item order populated in IDC_THEME above
+                default: theme = xaml::ElementTheme::Default; break
+                case 1: theme = xaml::ElementTheme::Light; break;
+                case 2: theme = xaml::ElementTheme::Dark; break;
+            }
+            auto panel = self->m_desktopWindowXamlSource.Content().as<controls::Panel>();
+            panel.RequestedTheme(theme);
```

Comment:
this can be simplified like this:

```suggestion
            self->m_theme = static_cast<xaml::ElementTheme>(ComboBox_GetCurSel(themeComboBox));
            auto panel = self->m_desktopWindowXamlSource.Content().as<controls::Panel>();
            panel.RequestedTheme(self->m_theme);
```